tfx/components/example_gen/component_test.py

```diff
@@ -129,9 +119,6 @@ def testConstructWithOutputConfig(self):
                      example_gen.outputs['examples'].type_name)
     artifact_collection = example_gen.outputs['examples'].get()
     self.assertEqual(1, len(artifact_collection))
-    self.assertEqual(['train', 'eval', 'test'],
```

Comment:
you can verify 
example_gen.exec_properties['input_config'] and example_gen.exec_properties['output_config'] to make sure it's set correctly

same for testConstructWithInputConfig

you don't need to add `self.assertEqual(['train', 'eval', xxx], generate_output_split_names(input_config, output_config))` in this test, but make sure example_gen/utils_test.py covers such input and output config combination for generate_output_split_names

Reply:
i see, i updated the component_test

also i checked -- utils_test.py will check for generate_output_split_names